GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -94,11 +98,20 @@
     GVRActivity mActivity;
     private int mCurrentEye;
 
+	private boolean mScreenshotCenterEnabled = false;
+	private boolean mScreenshotLeftEnabled = false;
+	private boolean mScreenshotRightEnabled = false;
+	private GVRScreenshotCallback mScreenshotCenterCallback = null;
+	private GVRScreenshotCallback mScreenshotLeftCallback = null;
+	private GVRScreenshotCallback mScreenshotRightCallback = null;
+
     private native void renderCamera(long appPtr, long scene, long camera,
             long renderTexture, long shaderManager,
             long postEffectShaderManager, long postEffectRenderTextureA,
             long postEffectRenderTextureB);
-
+    
+	private native void readbackResult(long renderTexture, Object readbackBuffer);
```

Comment:
This is a relatively minor issue, but "readbackResult" isn't a very descriptive name. "getScreenCapture" or "readRender" might be better.